master/docs/manual/configuration/buildsteps.rst

```diff
@@ -2183,6 +2183,9 @@ This step takes the following arguments:
 ``defines``
    (optional) Dictionary of defines to overwrite values of the :file:`conf.py` file.
 
+``strict_warnings``
+   (optional) Convert all warnings to errors.
```

Comment:
Could we reword to something like this so that information about the accepted type is present?

"If set to `True`, converts all warnings to errors (`-E` option). Defaults to `False`.